openstack/ironic/ci/test-values.yaml

```diff
@@ -1,7 +1,10 @@
 global:
+  registry: keppel.regionOne.cloud
+  dockerHubMirrorAlternateRegion: myRegistry/dockerhubalternate
   master_password: topSecret
   dbPassword: secret
   ipmi_exporter_user_passwd: topSecret
```

Comment:
Would it be possible to clean this up and either only use underscores or camelCase variables in the charts?

Reply:
I think that depends on what the chart requires and not what the tests provide? So not sure I have the power to change this